src/components/organisms/PaymentModal/PaymentForm.tsx

```diff
@@ -1,215 +0,0 @@
-import React, { useState, useEffect } from "react";
-import { useStripe, useElements, CardElement } from "@stripe/react-stripe-js";
-
-import { VenueEvent } from "types/venues";
-
-import CardInput from "components/molecules/CardInput";
-import firebase from "firebase/app";
-import "firebase/functions";
-import TabNavigation from "components/molecules/TabNavigation";
```

Comment:
Is `components/molecules/TabNavigation` still used anywhere? Not saying we should necessarily remove it regardless, if it looks like useful code. But curious.

Reply:
We can use it later
But no
nowhere is used after stripe delete